src/metabase/pulse/render/body.clj

```diff
@@ -214,30 +214,29 @@
   {:arglists '([chart-type render-type timezone-id card dashcard data])}
   (fn [chart-type _ _ _ _ _] chart-type))
 
-(defn- filter-rows [data table-columns rows]
-  (let [column-order      (qp.streaming/export-column-order (:cols data) table-columns)
-        keep-filtered-idx (fn [row] (if column-order
-                                      (let [row-v (into [] row)]
-                                        (for [i column-order] (row-v i)))
-                                      row))
-        filtered-rows     (map #(update % :row keep-filtered-idx) rows)]
-    filtered-rows))
+(defn- filter-rows [data viz-settings]
```

Comment:
should this be named something different if it's filtering & ordering both rows and cols?